weldx/core.py

```diff
@@ -496,7 +484,7 @@ def _interp_time_expression(
             data = np.expand_dims(data, 0)
 
         dax = xr.DataArray(data=data)  # don't know exact dimensions so far
-        return dax.rename({"dim_0": "time"}).assign_coords({"time": time_pd})
+        return dax.rename({"dim_0": "time"}).assign_coords({"time": time})
```

Comment:
Here is at least a partial repetition of the mentioned code above. Not sure if we can use it.

Reply:
Resolved in PR #445